burn-book/src/basic-workflow/backend.md

```diff
@@ -7,74 +7,21 @@ entrypoint of our program, namely the `main` function defined in `src/main.rs`.
 ```rust , ignore
 use burn::optim::AdamConfig;
 use burn::backend::{Autodiff, Wgpu, wgpu::AutoGraphicsApi};
-use guide::model::ModelConfig;
+use crate::model::ModelConfig;
 
 fn main() {
     type MyBackend = Wgpu<AutoGraphicsApi, f32, i32>;
     type MyAutodiffBackend = Autodiff<MyBackend>;
 
     let device = burn::backend::wgpu::WgpuDevice::default();
-    guide::training::train::<MyAutodiffBackend>(
+    crate::training::train::<MyAutodiffBackend>(
         "/tmp/guide",
-        guide::training::TrainingConfig::new(ModelConfig::new(10, 512), AdamConfig::new()),
+        crate::training::TrainingConfig::new(ModelConfig::new(10, 512), AdamConfig::new()),
         device,
     );
 }
 ```
 
-<details>
-<summary><strong>🦀 Packages, Crates and Modules</strong></summary>
```

Comment:
I guess it's less relevant now, but isn't it a bit sad to delete all the help about packages crates and modules? We could move it elsewhere and delete the parts about `guide`

Reply:
You are right, I was not sure where we could recycle it so I just removed it. That would be nice to find a place for it. Maybe in a dedicated page on how to run the examples ? For now we briefly mentions them at the start but maybe we should remove this mention and create a dedicated page at the end of the basic workflow as a way to explore more examples ?

Reply:
We still refer to the current crate with the changes, so perhaps most of the information can stay there and remains relevant 🤔 maybe just changing or removing the references to the `guide`.

Reply:
We need to reword it, I am not sure this is the best place for it though. I would put it where we are mentioning other examples because we do lib and examples mechanisms for them. Hence my proposal for a dedicated page for other examples where we can be more verbose and even list them with a small description (can be good for referencing).

Reply:
I ended up moving it to the get started page in the Examples paragraph.